samples/CommunityToolkit.Maui.Sample/Pages/Views/MediaElement/MediaElementPage.xaml.cs

```diff
@@ -237,33 +239,34 @@ async void ChangeAspectClicked(object? sender, EventArgs e)
 
 	void DisplayPopup(object sender, EventArgs e)
 	{
-		MediaElement.Pause();
-		MediaElement popupMediaElement = new MediaElement
```

Comment:
Sorry I don't understand this change, why is this now removing the current MediaElement and adding it into the Popup?

Reply:
Yes it removes it and adds it to popup. Before it just created a second MediaElement and added that. I can change it back and add a new media element and then remove it if that is preffered. I prefer to do this as it shows it is possible and not required. I have had issues with edge cases when creating a new media element. After closing popup disposing of media element is not done until page is closed. This causes issues if you click on popup again. With the changes above the issue is mitigated.

Reply:
> Sorry I don't understand this change, why is this now removing the current MediaElement and adding it into the Popup?

@bijington I updated it so that it now has the same behavior as before but without any crashes.

Reply:
Back to my original question though, is there anything we can change in the **library** rather than the sample application to prevent the crash for other developers?

Reply:
I'm waiting for media 3 to be ready and merged before deciding on this. I'm not having any issues with media 3 and this PR might be moot. But keeping it open until the PR is final and merged without issues. Current testing indicates this will be closed when media 3 is ready.

Reply:
OK thanks